packages/ui/src/views/repo/pull-request/details/components/conversation/sections/pull-request-merge-section.tsx

```diff
@@ -49,145 +91,103 @@ const PullRequestMergeSection = ({
       code: `git push origin ${pullReqMetadata?.source_branch}`
     }
   ]
-  const stepInfo = (item: { step: string; description: string; code?: string; comment?: string }, index: number) => {
-    return (
-      <div key={index}>
-        <Layout.Horizontal className="gap-x-1">
-          <h3 className="text-14 font-medium text-foreground-1">{item.step}</h3>
-          <Layout.Vertical className="w-[90%] max-w-full">
-            <p className="text-14 text-foreground-4">{item.description}</p>
-            <p
-              className={cn('text-14 text-foreground-4 ', {
-                'border border-border rounded-md px-2 py-1 !my-2': item.code,
-                '!my-1': item.comment
-              })}
-            >
-              <Layout.Horizontal className="items-center justify-between">
-                <p>{item.code ? item.code : item.comment}</p>
-                {item.code && <CopyButton name={item.code} />}
-              </Layout.Horizontal>
-            </p>
-          </Layout.Vertical>
-        </Layout.Horizontal>
-      </div>
-    )
-  }
 
-  const onValueChange = (value: string | string[]) => {
-    if (typeof value === 'string') {
-      setValue(value)
-    }
+  const handleCommandLineClick = (e: MouseEvent<HTMLButtonElement>) => {
+    e.stopPropagation()
+
+    setAccordionValues(prevState => [...prevState, ACCORDION_VALUE])
+    setShowCommandLineInfo(!showCommandLineInfo)
```

Comment:
For toggling state, it's usually a good idea to use the callback version
```suggestion
    setShowCommandLineInfo(current => !current)
```